apps/course/src/app/shared/navigation/navbar-course/navbar-course.component.html

```diff
@@ -40,7 +40,7 @@
         </ul>
       }
     </div>
-    <a class="btn btn-ghost text-xl" routerLink="/">
+    <a class="btn btn-ghost text-xl" href="javascript:history.back()">
```

Comment:
Hmm, not really. The back button logic should be more complex, since when I switch in a course `c/:courseId/` to `c/:courseId/agenda` and then press the back button I get routed to `c/:courseId/` again and not the page I was before I entered a course detail.

Reply:
Yes it does do it.

Reply:
Well the back button should, as described in the issue, route back to the page I visited all before I was on the course detail itself. So no matter where I go on the course detail and then press the back button I go back to f.x. the course overview since I entered the course detail from there originally.

https://github.com/danieljancar/chorizo/assets/112062588/1e6d27ea-d740-46c3-af00-fa426c036cc8

Reply:
I asked Andrea and it goes back to the page you were before, like you asked. I don't get what else you want. Could you please explain it some more? I really don't get it and neither does Andrea.

Reply:
Scenario expected:
1. I open the website at root
2. Navigate to course overview at `c`
3. Open a course detail `c/:courseId/``
4. In the course detail navigate to `c/:courseId/agenda`
5. And when I press the "leave course" button now I get routed to course overview again at `c` and not `c/:courseId/`

Scenario your current implementation:
1. I open the website at root
2. Navigate to course overview at `c`
3. Open a course detail `c/:courseId/``
4. In the course detail navigate to `c/:courseId/agenda`
5. And when I press the "leave course" button button I get navigated to `c/:courseId` and not the route I was before I entered the course detail

Reply:
I changed it. 
I hope that this is what you mean.